lint/flake8.bzl

```diff
@@ -76,14 +76,16 @@ def _flake8_aspect_impl(target, ctx):
     if not should_visit(ctx.rule, ctx.attr._rule_kinds):
         return []
 
-    report, exit_code, info = report_files(_MNEMONIC, target, ctx)
+    outputs, info = output_files(_MNEMONIC, target, ctx)
 
     files_to_lint = filter_srcs(ctx.rule)
 
     if len(files_to_lint) == 0:
-        dummy_successful_lint_action(ctx, report, exit_code)
+        dummy_successful_lint_action(ctx, outputs.human.stdout, outputs.human.exit_code)
     else:
-        flake8_action(ctx, ctx.executable._flake8, files_to_lint, ctx.file._config_file, report, exit_code)
+        flake8_action(ctx, ctx.executable._flake8, files_to_lint, ctx.file._config_file, outputs.human.stdout, outputs.human.exit_code)
+
+    flake8_action(ctx, ctx.executable._flake8, files_to_lint, ctx.file._config_file, outputs.machine.stdout, outputs.machine.exit_code)
```

Comment:
same comment for flake8 as for shellcheck, https://github.com/aspect-build/rules_lint/pull/331/files#r1680423092